src/reducers/chartReducer.js

```diff
@@ -124,28 +121,48 @@ export const chartWindowAction = (
             type: CHART_WINDOW,
             windowBegin: 0,
             windowEnd: 0,
-            windowDuration: windowDuration === null ? null : duration,
+            windowDuration,
             yMin: y0,
             yMax: y1,
         };
     }
-    const half = duration / 2;
+    const half = windowDuration / 2;
     const center = (windowBegin + windowEnd) / 2;
     return {
         type: CHART_WINDOW,
         windowBegin: center - half,
         windowEnd: center + half,
-        windowDuration: duration,
+        windowDuration,
         yMin: y0,
         yMax: y1,
     };
 };
 
-export const chartReset = windowDuration =>
```

Comment:
Why remove this? It was quite nice imo, with its descriptive name,

Reply:
Because it wasn't used.